build/commands/scripts/sync.js

```diff
@@ -251,6 +251,7 @@ async function RunCommand() {
   Log.progress('...gclient sync done.')
 
   await util.applyPatches()
+  util.updateChromeVersion()
```

Comment:
Would it be useful to log the updated version number?

Reply:
> Would it be useful to log the updated version number?

added:
```
...
Done applying patches.
chrome/VERSION: 115.1.55.16
Running gclient runhooks...
```

Reply:
don't we need to ignore this now when we update patches?

Reply:
> don't we need to ignore this now when we update patches?

this PR explicitly does not exclude `chrome/VERSION` from patch machinery to not break our current CI automation. When this will be merged, we should uplift it to beta, release and then remove `chrome-VERSION.patch` modification from CI scripts.

Reply:
@petemill will this cause any issues with the logic that skips the chromium src sync for minor version changes? I don't think it should, but just double checking

Reply:
Yes I'm wondering about that in this comment https://github.com/brave/brave-core/pull/19062#discussion_r1259980656